extmod/modlwip.c

```diff
@@ -1033,7 +1033,7 @@ STATIC mp_obj_t lwip_socket_sendall(mp_obj_t self_in, mp_obj_t buf_in) {
             break;
         }
         case MOD_NETWORK_SOCK_DGRAM:
-            mp_not_implemented("");
+            mp_raise_NotImplementedError("");
```

Comment:
Thanks for this. I did it because I thought the reason for those functions was code style unification. I'll take the advice for any future patches.